services/httpd/handler.go

```diff
@@ -257,23 +257,17 @@ func (h *Handler) serveQuery(w http.ResponseWriter, r *http.Request, user *meta.
 	p := influxql.NewParser(strings.NewReader(qp))
 	db := q.Get("db")
 
+	// Sanitize the request query params so it doesn't show up in the response logger.
+	// Do this before anything else so a parsing error doesn't leak passwords.
+	sanitize(r)
```

Comment:
Since `query.Statements` is no longer used, I think `sanitize` can be called in the response logger.

https://github.com/influxdata/influxdb/blob/master/services/httpd/response_logger.go#L94